compiler/src/dotty/tools/backend/sjs/JSCodeGen.scala

```diff
@@ -2815,9 +2815,179 @@ class JSCodeGen()(using genCtx: Context) {
             js.ForIn(objVarDef.ref, keyVarIdent, NoOriginalName, {
               js.JSFunctionApply(fVarDef.ref, List(keyVarRef))
             }))
+
+      case REFLECT_SELECTABLE_SELECTDYN =>
+        // scala.reflect.Selectable.selectDynamic
+        if (currentClassSym.get == jsdefn.ReflectSelectableClass)
+          genNormalApply(tree, isStat)
```

Comment:
Why do we need these ifs? I assume the compiler doesn't emit code for primitives (or I hope so :P).

Reply:
Good point. Primitives were emitted, and this is why I needed those. Fixed now.